tests/features/upgradeSim.spec.ts

```
@@ -0,0 +1,264 @@
+// Creates representative dashboards and saves their UIDs
+// to tests/fixtures/ for phase2 to consume.
+// ─────────────────────────────────────────────────────────
+
+test('phase1: v0.3.1 provisioned dashboards load without JS crash', async ({
+  readProvisionedDashboard,
+  page,
+}) => {
+  for (const version of ['v0.2.2', 'v0.2.3', 'v0.3.0']) {
+    const dashboard = await readProvisionedDashboard({ fileName: `${version}.json` });
+    const errors: string[] = [];
+    page.on('console', (m) => { if (m.type() === 'error') errors.push(m.text()); });
```

ESLint

[error] Expected { after 'if' condition.

(curly)